gfop/get_food_counts.py

```diff
@@ -15,19 +15,19 @@ def load_food_metadata():
     return gfop_metadata
 
 
-def get_sample_types(simple_complex=None):
+def get_sample_types(simple_complex='all'):
     """
     Return:
         Global FoodOmics ontology containing only simple, only complex, or all foods.
     Args:
-        simple_complex (string): one of 'simple', 'complex', or 'None'.
+        simple_complex (string): one of 'simple', 'complex', or 'all'.
                                  Simple foods are single ingredients while complex foods contain multiple ingredients.
-                                 'None' will return both simple and complex foods.
+                                 'all' will return both simple and complex foods.
     """
     gfop_metadata = load_food_metadata()
-    if simple_complex is not None:
+    if simple_complex is not 'all':
```

Comment:
`is` checks whether it's the same object, `==` checks whether it's the same value. So the `is not` should be replaced by `!=`.